draft-ietf-lake-edhoc.md

```diff
@@ -315,15 +315,15 @@ In order to create a "full-fledged" protocol some additional protocol elements a
 
 * A keying material exporter and a key update function with forward secrecy.
 
-* Verification of the selected cipher suite.
+* Negotiation of cipher suite, more specifically verification by Responder of most preferred cipher suite by Initiator
```

Comment:
"more specifically verification by Responder of most preferred cipher suite by Initiator"

Not sure this makes thing easier to understand. This sentence is not correct, but a correct sentence would be even longer and complicated....

Reply:
Abbreviated, see 67947ff below.